ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/receive/view/ReceiveView.kt

```diff
@@ -170,73 +162,139 @@ private fun ReceiveContents(
     versionInfo: VersionInfo,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier =
-            Modifier
-                .fillMaxHeight()
-                .verticalScroll(rememberScrollState())
```

Comment:
We still need to use these modifiers to keep the below content vertically scrollable.